source/digits_hits/src/GateRestrictedVolumeMessenger.cc

```diff
@@ -0,0 +1,58 @@
+/** ----------------------
+  Copyright (C): OpenGATE Collaboration
+  This software is distributed under the terms
+  of the GNU Lesser General  Public Licence (LGPL)
+  See LICENSE.md for further details
+  ----------------------*/
+
+
+#include "GateRestrictedVolumeMessenger.hh"
+
+#include "GateRestrictedVolume.hh"
+
+#include "G4UIcmdWithAString.hh"
+
+GateRestrictedVolumeMessenger::GateRestrictedVolumeMessenger(GateRestrictedVolume* itsRestrictedVolume)
+    : GatePulseProcessorMessenger(itsRestrictedVolume)
+{
+  G4String guidance;
+  G4String cmdName;
+  G4String cmdName2;
+
+  cmdName = GetDirectoryName() + "setRestrictedVolume";
+  restrictVolCmd = new G4UIcmdWithAString(cmdName,this);
+  restrictVolCmd->SetGuidance("Set restricted volume for pulse-discrimination");
+
+  cmdName2 = GetDirectoryName() + "setRestrictionMode";
+  restrictModeCmd = new G4UIcmdWithAString(cmdName2,this);
+  restrictModeCmd->SetGuidance("Set restriction mode, two are available: reject accept");
+
+}
+
+GateRestrictedVolumeMessenger::~GateRestrictedVolumeMessenger()
+{
+  delete restrictVolCmd;
+  delete restrictModeCmd;
+}
+
+void GateRestrictedVolumeMessenger::SetNewValue(G4UIcommand* command, G4String newValue)
+{
+  if (command==restrictVolCmd) { RestrictedVolume()->SetRestrictedVolume(newValue); }
+  
+  else if (command==restrictModeCmd) { 
+  
+    if (newValue == "accept" || newValue == "reject")
+      RestrictedVolume()->SetRestrictionMode(newValue);
+       
```

Comment:
```suggestion
    if (newValue == "accept" || newValue == "reject") {
      RestrictedVolume()->SetRestrictionMode(newValue);
    }       
```
Just for better reading.

Reply:
I reformat with clang-format once again

Reply:
I am not sure how to squash and force-push commits.

Reply:
Please use ``git rebase -i HEAD~<number of commits> (currently 4)``, then select ``f`` (fixup) for all commits but first one and upload new version using same way you pushing, but adding ``--force`` flag, e.g.(please adapt upstream repository name): ``git push upstream generic_module_issue --force``.

More information about rebase and squash is available at: https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

Reply:
I rebased the code